### PR TITLE
Changed MainWindow title

### PR DESCRIPTION
**Pull request by RostyslavGarbar**

### CODE REVIEW

File: MainWindow.xaml [hunk 1]
```diff
@@ -5,7 +5,7 @@
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:GeekGrapher"
         mc:Ignorable="d"
-        Title="MainWindow" Height="450" Width="800">
+        Title="RostykHarbar" Height="450" Width="800">
```
Comment (SviatoslavBachynskyi) [inline MainWindow.xaml]: you are not the only one creator of this project, change title to Geek Grapher

**Review by SviatoslavBachynskyi — CHANGES_REQUESTED** [review]
please, read all the comments and address requested changes

**Review by SviatoslavBachynskyi — APPROVED** [review]
Good job!

**Comment by RostyslavGarbar** [comment]
Changed MainWindow title

**Comment by RostyslavGarbar** [comment]
Changed MainWindow title

**Comment by RostyslavGarbar** [comment]
Changed MainWindow title

**Pull request merged**